lib/syskit/dynamic_port_binding.rb

```diff
@@ -203,6 +203,7 @@ def initialize(
             # Method called by {Accessor} to create the accessor object from a
             # port
             def create_accessor(port)
+                @policy.merge(init: @policy.fetch(:init, port.model.init_policy?))
```

Comment:
- `merge` is returning a new hash. This line does nothing.
- the best way to control priority with `merge` is to decide what hash gets merged into which, essentially: `low_priority.merge(high_priority)`